dataprep/data_connector/implicit_database.py

```diff
@@ -54,6 +54,7 @@ class ImplicitTable:  # pylint: disable=too-many-instance-attributes
     body_ctype: str
     body: Optional[Fields] = None
     cookies: Optional[Fields] = None
+    pag_params: Dict[str, str]
```

Comment:
Because pagination parameters are already clearly defined and deterministic, please change it to `pag_params: Optional[Pagination]` and define your own `class Pagination` to make the code clear.